.circleci/postgres/docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: '3.3'
+services:
+  cdash:
+    image: "kitware/cdash:testing"
+    container_name: cdash
+    build:
+      context: .
+      dockerfile: ./docker/cdash.docker
+      args:
+        - CDASH_DB_HOST=postgres
+        - DEVELOPMENT_BUILD=1
+    depends_on:
+      - postgres
+      - selenium-hub
+      - chrome
+    links:
+      - postgres
+    ports:
+     - "8080:80"
+    environment:
+      CDASH_ROOT_ADMIN_PASS: secret
+
+      CDASH_STATIC_USERS: |
+        USER jdoe@acme.com jdoe_secret
+          John Doe "ACME Inc."
+
+        ADMIN admin@example.org admin_secret
+          Admin User "Example Foundation"
+
+        donotreply@example.org oldsecret newsecret
+
+        strange.name@gmail.com strange_secret
+        INFO Str@nge N@me "Str@nge Rese@rch Gr0up LLC."
+
+        DELETE remove.this.user@example.org delete_secret
```

Comment:
Do we need these users for testing on CircleCI? Generally speaking it looks like a lot of copy/paste between the MySQL and postgres config.

Reply:
Not sure what will happen if you remove them, and we can probably find some optimized solution between the two compose files. Given the larger grand scheme of things, maybe we can hold off on this particular change.